lib.ts

```diff
@@ -1,4 +1,4 @@
-import { Anchorage } from "./types";
+import { Anchorage, Position } from "./types";
 
 export const newAnchorage = (width: number, height: number): Anchorage => {
```

Comment:
if you follow the idea with `Vector2d`, you can refactor this method to accept `Dimensions` as parameter